python_modules/dagster/dagster/core/definitions/sensor_definition.py

```diff
@@ -224,12 +228,16 @@ def __init__(
                     ),
                 )
             ]
-        elif job:
```

Comment:
this case is redundant since `jobs` is set to `[job]` on line 198. removed to reduce code duplication, but I can add it back in if it makes the code more clear to read/understand

Reply:
reverting since this seems to be what was causing a pylint error in a seemingly unrelated code block